src/App.jsx

```diff
@@ -14,70 +15,74 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const NO_GOODS = 'No goods selected';
+  const [value, setValue] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {(value !== NO_GOODS)
+          ? (
+            <>
+              {`${value} is selected`}
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+              <button
+                onClick={() => {
+                  setValue(NO_GOODS);
+                }}
```

Comment:
```suggestion
                onClick={reset}
```
Consider moving this function from jsx to a separate function